src/inmanta/app.py

```diff
@@ -114,9 +115,15 @@ def _handle_startup_done(fut: asyncio.Future) -> None:
 def start_agent(options: argparse.Namespace) -> None:
     from inmanta.agent import agent
 
-    util.ensure_event_loop()
+    # The call to configure() should be done as soon as possible.
+    # If an AsyncHTTPClient is started before this call, the max_client
+    # will not be taken into account.
+    max_clients: int = Config.get("agent_rest_transport", "max_clients", "10")
+    AsyncHTTPClient.configure(None, max_clients=max_clients)
```

Comment:
On second thought, perhaps we should only set this when the config value is explicitly set? i.e. use `None` as default and only call `configure` if not `None`? Wdyt?